README.md

```diff
@@ -133,14 +133,16 @@ Add package.json scripts
 
 Run `npm run cosmos` or `yarn cosmos` and go to [localhost:8989](http://localhost:8989) 🎉
 
-> If you rely on the default webpack config, make sure to install the Babel and webpack plugins yourself. Depending on your needs, you'll probably want `babel-preset-env babel-preset-react babel-loader style-loader css-loader html-webpack-plugin`. Finally, add `.babelrc` to your project root.
+> If you rely on the default webpack config, make sure to install the Babel and webpack plugins yourself. Depending on your needs, you'll probably want `@babel/core @babel/preset-env @babel/preset-react babel-loader style-loader css-loader html-webpack-plugin`. Finally, add `.babelrc` to your project root.
 >
 > ```
 > {
->   "presets": ["env", "react"]
+>   "presets": ["@babel/env", "@babel/react"]
 > }
 > ```
 
+_NOTE: Babel v8 works differently than v7 and the package naming conventions changed. The above steps and Babel config are for the latest v8 version._
```

Comment:
A few notes:
- Please put this note inside the quote above, to keep mentions of 3rd party deps together
- Babel packages are namespaces since 7.x, not 8.x (which doesn't exist, only for babel-loader which is webpack related)
- You can say something like "This example applies for Babel 7.x and above..."
- You may want to link to https://babeljs.io/docs/en/v7-migration as well

Reply:
Ah, thanks for the catch on the Babel version. The major version on babel-loader threw me off. I'll make these changes now.

I was trying to figure out why you had that quote block! Makes complete sense now.